controllers/token.go

```diff
@@ -28,8 +29,14 @@ import (
 // @router /get-tokens [get]
 func (c *ApiController) GetTokens() {
 	owner := c.Input().Get("owner")
+	limit := util.ParseInt(c.Input().Get("pageSize"))
+	page := util.ParseInt(c.Input().Get("page"))
+	if limit <= 0 || page < 1 {
+		c.ResponseError("Page or limit params error")
+		return
+	}
 
-	c.Data["json"] = object.GetTokens(owner)
+	c.ResponseOk(object.GetTokens(owner, limit, (page-1)*limit))
```

Comment:
?

Reply:
Here need to return array of current page and total number of records. But `c.Data["json"]` only need one interface param.